BatterySensorDataDisplay.hpp

```diff
@@ -0,0 +1,44 @@
+
+#include <iostream>
+#include "BatterySensor.hpp"
+
+class displaySensorDataInterface
+{
+    public:
+        virtual void displayDataOnConsole(std::string data)=0;
+        virtual std::string displayFormattedData(unsigned int SOC, signed int temp)=0;
+};
+
+class displaySensorData:public displaySensorDataInterface
+{
+    public:
+        void displayDataOnConsole(std::string data)
+        {
+            std::cout<<data<<std::endl;
+        }
+        
+        std::string displayFormattedData(unsigned int SOC, signed int temp)
+        {
+            std::string formattedData;
+            formattedData=std::to_string(SOC)+","+std::to_string(temp);
+            return formattedData;
+        }
+};
+
+class mockSensorData:public displaySensorDataInterface
+{
+    public:
+        int displayCounter=0,formatCounter=0;
+        
+        void displayDataOnConsole(std::string /*data*/)
+        {
+            displayCounter++;
+        }
+        
+        std::string displayFormattedData(unsigned int /*SOC*/,signed int /*temp*/)
+        {
+            formatCounter++;
+            return "";

```

Comment:
Better to return some characters instead of empty string

Reply:
Since the output was in the comma separated values format, other characters were not included.